docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/05 - Visits.md

```diff
@@ -0,0 +1,44 @@
+# Visits
+
+## Overview
+The **Visit** designates the instance in time where a candidate has visited or contacted a site and underwent any sort of data collection.
+
+**Visit** and **Timepoint** are terms used interchangeably in LORIS. **Visits** are defined in the `Visit_Windows` table of the database.
+
+## Adding Options
+
+### Front End
+ _not yet available_
+
+### SQL (Recommended)
+
+This table must be populated with visit labels - the Imaging Pipeline critically depends on this.  To populate with visit labels, run tools/populate_visit_windows.php - or manually insert study-specific information:
+
+```sql
+INSERT INTO Visit_Windows (Visit_label,  WindowMinDays, WindowMaxDays, OptimumMinDays, OptimumMaxDays, WindowMidpointDays) VALUES ('V1', '0', '100', '40', '60', '50');
```

Comment:
This document should probably describe what the Visit_Windows is used for a little more before telling people to populate it.

It's used to flag the Candidate Age as an error/warning when an instrument is administered, with "Optimum" determining when the study protocol says it's supposed to be done, and "Window" being when it becomes an error.

Reply:
Agreed the definition/function of windows needs to be clarified before we ask users to set them.  As discussed with @ridz1208 this week, there have been several conversations about this in the last year-ish, there's always confusion.  

See attached photos of what user sees on frontend/relevant notes.  I need help determining exactly what these windows do (someone to check the code) so I can write clear explanation to insert in this Wiki & propose frontend GUI changes to make this feature clear to users (i.e. label tweaks for clarity, better display of warning when in instrument_list, etc.). 

![config_windows](https://user-images.githubusercontent.com/31326609/53266706-9e07d180-36af-11e9-99a8-4976c50010b4.jpg)
![windows_instrument_view_test-loris](https://user-images.githubusercontent.com/31326609/53266708-9f38fe80-36af-11e9-8b77-87e3dfb34d1d.JPG)

Also, if I am correct in assuming how feature is meant to work, it appears to be broken on test. loris.